cluster/common.sh

```diff
@@ -80,7 +80,9 @@ function create-kubeconfig() {
   fi
 
   "${kubectl}" config set-cluster "${CONTEXT}" "${cluster_args[@]}"
-  "${kubectl}" config set-credentials "${CONTEXT}" "${user_args[@]}"
+  if [ -n "${user_args[@]:-}" ]; then
```

Comment:
Nit: per our chosen bash style guide [[ is prefered over [

https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/devel/coding-conventions.md